music/package.json

```diff
@@ -8,14 +8,15 @@
   "jsdelivr": "dist/magentamusic.js",
   "unpkg": "dist/magentamusic.js",
   "dependencies": {
-    "@tensorflow/tfjs": "^1.1.2",
+    "@tensorflow/tfjs": "^2.7.0",
```

Comment:
have you double checked everything else works when we switch to 2.x? Like, are all the tests passing, etc?

Reply:
Not yet, but definitely something I am planning to do.